internal/xds/bootstrap/bootstrap.go

```diff
@@ -36,6 +36,9 @@ const (
 	envoyReadinessAddress = "0.0.0.0"
 	EnvoyReadinessPort    = 19001
 	EnvoyReadinessPath    = "/ready"
+
+	// DefaultEnvoyStatsMatcherPrefixes is the default subset of Envoy stats added when setting StatsMatcher.
```

Comment:
why is this needed ?

Reply:
There are two reasons：

1. [disable cluster manager stats will block the cds](https://github.com/envoyproxy/envoy/issues/28179), so cluster_manager stats is is required；
2. I think some basic stats are necessary，users may not be aware of these stats, but it is very necessary in the actual troubleshooting process.